cloud_process:
  planar_threshold: 0.01
  x_min: -1.1
  y_min: -1.1
  z_min: 0.01
  x_max: 1.1
  y_max: 1.1
  z_max: 1.01